### DISCUSSION TITLE
.is-a.dev Domain Issue

### DISCUSSION THREAD

**Discussion by Sidd-underscore** [vercel/community#5825] [Help] [answered]
### Summary

Hello Vercel Community! Thank you so much for reading this. 

I am trying to register the domain "sidd.is-a.dev" on https://vercel.com/sidd-underscore/sidd-website. "is-a.dev" is a free domain system that grants anyone access to their subdomain of choice. Therefore, other "*.is-a.dev" domains exist, and Vercel says "Another Vercel account is using this domain. Please set the following TXT record on _vercel.is-a.dev to use sidd.is-a.dev in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed." Of course, I don't know who the first person to own an "is-a.dev" domain is, so how do I avoid this?

Thank you!

PS: the only other solution that I could find was answered by a staff member saying they answered the question in a private ticket, so this is why I am re-asking this question :)

### Steps to Reproduce

1. Assign basically any .is-a.dev domain to any project
2. Oh no! You have to verify!

**Comment by TooTallNate** [comment, marked as answer]
In order for this to work the way you're expecting, the `is-a.dev` domain would need to be submitted to the [Public Suffix List](https://publicsuffix.org/).

**Reply by bestZwei** [reply]
it has been added to Public Suffix List, but I still cannot add it to vercel
